### DISCUSSION TITLE
Extending "customizable block text style"

### DISCUSSION THREAD

**Discussion by sfederici** [ScratchAddons/ScratchAddons#6959] [Ideas]
The idea behind the "customizable block text style" is really excellent. 

To make the addon really useful, it is necessary to extend the custom text size to
- string listed in the menus of the blocks
- dialogs appearing when using block related features, e.g. the "new message" dialog

**Comment by Secret-chest** [comment]
> * dialogs appearing when using block related features, e.g. the "new message" dialog

Doesn't make much sense, because it would be inconsistent with the others. Also there is no new message dialog.

**Reply by sfederici** [reply]
> there is no new message dialog.

Open the menu in the "when i receive" block then select "New Message".

**Comment by WorldLanguages** [comment]
I guess you mean this, by the first item:

![image](https://github.com/ScratchAddons/ScratchAddons/assets/17484114/38884472-76cd-47d7-a6f0-1526be443b97)

That probably deserves an issue of its own

**Reply by sfederici** [reply]
Yes, this for the first item. And this

<img width="369" alt="immagine" src="https://github.com/ScratchAddons/ScratchAddons/assets/2651868/32ee1828-68fc-407a-a543-b5f52cf04fbc">

for the second one. I'm planning to use this addon so my students can read better in the projector screen without me having to zoom everything too much. But if some characters are the right size and other ones are smaller (like, in this example, the name of the message that I'm going to write, or the options in the menus), this addon is not going to be useful to me

**Reply by WorldLanguages** [reply]
Also editor comments: #6881
